pylab/website/tests/test_events.py

```diff
@@ -20,6 +20,11 @@ def test_event_details_page(self):
             osm_map_link='http://openstreetmap.org/',
             description='Test description',
         )
-        resp = self.app.get('/events/2015/07/29/test-title/')
+
+        resp = self.app.get('/events/2015/07/29/test-title/', user=AnonymousUser())
+
+        self.assertEqual(resp.status_int, 200)
+
+        resp = self.app.get('/events/2015/07/29/test-title/', user=user)
 
```

Comment:
You should add a test for functionality you just created. E.g. test this use case: user opens event-detail page, selects attendance response, saves it and sees that his response is correctly presented in the attendances list in that event-details page.